docker/env/hydra.sh

```diff
@@ -248,6 +248,7 @@ function run_in_docker () {
         -v /tmp:/tmp \
         -v /var/tmp:/var/tmp \
         -v "${HOME_DIR}:${HOME_DIR}" \
+        -v "/tmp/.local-${SCT_TEST_ID}:${HOME_DIR}/.local" \
```

Comment:
why no use tmpfs ?

Reply:
You can pick up any other dir if you want.